psi4/driver/procedures/proc.py

```diff
@@ -2172,8 +2172,14 @@ def run_ccenergy(name, **kwargs):
     if core.get_global_option('RUN_CCTRANSORT'):
```

Comment:
I think I agree on the second option. Seems more extensible. And options are bad enough without overwriting them. Maybe stick a commented `USING_transqt2` near the option in read_options.cc, as a guide to searching.